src/core/interfaces.ts

```diff
@@ -72,4 +72,7 @@ export interface Entity<C extends Component> {
   selection: d3.Selection<any>;
   component: C;
 }
+
+export type Numeric = number | {valueOf(): number};
```

Comment:
I'm guessing the point of having this type is to accept number or Date objects? I dug around a lot but couldn't find a clear answer.